src/platforms/ios/ioscontroller.swift

```diff
@@ -188,10 +188,21 @@ public class IOSControllerImpl : NSObject {
                 if let error = saveError {
                     IOSControllerImpl.logger.error(message: "Connect Tunnel Save Error: \(error)")
                     failureCallback()
+                    if isOnboarding {
+                        IOSControllerImpl.logger.error(message: "Finishing onboarding...")
```

Comment:
```suggestion
                        IOSControllerImpl.logger.error(message: "Finishing onboarding, but saving tunnel resulted in error...")
```

Reply:
Do you think this clarification is still needed after the error log 3 lines above that says "Connect Tunnel Save Error"?

Reply:
Personally, I think it makes it even clearer when a future person is skimming logs, but I don't feel strongly about that at all - I'm okay if you want to keep it as is.